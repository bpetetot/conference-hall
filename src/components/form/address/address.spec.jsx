/* eslint-env jest */
import { shallow } from 'enzyme'
import React from 'react'
import AddressInput from './index.js'


describe('components/address', () => {
  it('should render with children', () => {
    expect(shallow(<AddressInput value="value" onChange={() => {}} />)).toMatchSnapshot()
  })
})
