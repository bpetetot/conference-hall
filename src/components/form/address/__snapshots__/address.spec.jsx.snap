// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/address should render with children 1`] = `
<PlacesAutocomplete
  debounce={200}
  highlightFirstSuggestion={false}
  onChange={[Function]}
  onError={[Function]}
  searchOptions={Object {}}
  shouldFetchSuggestions={true}
  value="value"
>
  <Component />
</PlacesAutocomplete>
`;
